guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -0,0 +1,30 @@
+[id="transport-modes-for-remote-execution_{context}"]
+= Transport Modes for Remote Execution
+
+You can configure your {Project} to use two different modes of transport for remote job execution.
+
+By default, remote execution transports job details over SSH.
+This is referred to as _push mode_.
+The SSH service must be enabled and active on the target hosts and the remote execution {SmartProxy} must have access to port 22 on the target hosts.
```

Comment:
Can you set the default port somewhere (global settings, host groups using parameters?) Is it always 22?

Reply:
You can, but then it is up to you as a user to ensure that the target machines have a ssh server listening on that custom, nonstandard port

Reply:
Thanks for clarifying. I'll leave it up to you to potentially add a small note that this depends on a setting.